packages/messages/src/browser/notifications.ts

```diff
@@ -51,6 +55,10 @@ export class Notifications {
         this.container.appendChild(notificationElement);
     }
 
+    create(properties: NotificationProperties): ProgressMessage {
+        return new ProgressNotification(this.container, this.createNotificationElement(properties), this.oncCancel, properties);
```

Comment:
1. it's not a good idea to share a single cancel `Event` between multiple `ProgressMessage` instances? Have you tried? 

2. naming of `create(properties: NotificationProperties): ProgressMessage`: I wouldn't expect this to return a `ProgressMessage`